### DISCUSSION TITLE
Error when try to update from 1.3

### DISCUSSION THREAD

**Discussion by Tigerliljan** [kimai/kimai#3818] [On-Premise: Installation & Updates]
When trying to update from 1.3.1 to latest I get stucket when "git checkout 1.30.5" I get this response "pathspec '1.30.5' did not match any file(s) known to git"
Any idea how to solve?

**Comment by kevinpapst** [comment]
What is 1.3.1? Does not exist: https://github.com/kimai/kimai/releases?page=8

Are you talking about https://www.kimai.org/documentation/migration-v1.html ?

**Comment by Tigerliljan** [comment]
1.3.1 is the version I can see in version_row in  DB
I'm still struggling to update the php.
Is it possible to create a fresh install and connect to existing DB (or is there also DB updates?)

**Reply by kevinpapst** [reply]
You will not get further help if you ignore clear questions.

**Reply by Tigerliljan** [reply]
I'm not really sure which question I missed?
I checked the version.php and it also have version 1.3.1

**Reply by kevinpapst** [reply]
You cannot upgrade. 
You are using the ancient Kimai 1.
You have to install (for you: the new) Kimai and then migrate your data as explained under the link I posted.

**Reply by Tigerliljan** [reply]
Thanks, then I know!
